newIDE/app/src/ExportAndShare/GenericExporters/OnlineWebExport/OnlineGameLink.js

```diff
@@ -303,37 +304,44 @@ const OnlineGameLink = ({
               open
               onRequestClose={() => setIsShareDialogOpen(false)}
               onApply={() => {
-                if (game && buildUrl && !isBuildPublished) {
+                if (game && buildOrGameUrl && !isBuildPublished) {
                   setIsOnlineGamePropertiesDialogOpen(true);
                 }
               }}
               flexColumnBody
             >
-              {buildUrl && !isGameLoading ? (
+              {buildOrGameUrl && !isGameLoading ? (
                 <ColumnStackLayout noMargin>
-                  <ShareLink url={buildUrl} />
-                  {isBuildPublished && (
-                    <ColumnStackLayout noMargin expand>
-                      {navigator.share ? (
-                        <ShareButton url={buildUrl} />
-                      ) : (
-                        <Column
-                          expand
-                          justifyContent="flex-end"
-                          noMargin
-                          alignItems="flex-end"
-                        >
-                          <SocialShareButtons url={buildUrl} />
-                        </Column>
-                      )}
-                      <GameRegistration
-                        project={project}
-                        hideLoader
-                        suggestAdditionalActions
-                      />
-                    </ColumnStackLayout>
-                  )}
-                  {!isBuildPublished && game && (
+                  <ShareLink url={buildOrGameUrl} />
+                  <ColumnStackLayout noMargin expand>
+                    {navigator.share ? (
+                      <ShareButton url={buildOrGameUrl} />
```

Comment:
You changed the logic a bit here to display the share buttons even for instant builds.
Is that on purpose?
I think it's fine, just checking

Reply:
Yes, I did it on purpose: even if the link is private, you might want to send it to someone else or yourself